src/main/java/com/fasterxml/jackson/annotation/JsonTypeInfo.java

```diff
@@ -216,7 +216,7 @@ public enum As {
 =======
          *
          * @since 2.3 but databind <b>only since 2.5</b>.
->>>>>>> 2.16
+>>>>>>> 2.16 
```

Comment:
Hmmh. My mistake, shouldn't have left these around. For 3.0 all "@since 2.x" notes to be removed (assumption being that anything in 3.0 or earlier does not need it).... although now wondering if same should apply to annotations package or not.

But for now, yeah let's remove all `@since`s (except maybe in some places can add `@since 3.0` for something introduced only in 3.x.